File: 1. Sum of cubes/code.cpp
#include <iostream>
using namespace std;

int sumOfCubes(int n, int m) {
    int total_sum = 0;
    for (int i = n; i <= m; ++i) {
        total_sum += i * i * i;
    }
    return total_sum;
}

int main() {
    int n = 2, m = 4;
    cout << sumOfCubes(n, m) << endl; // Output should be 99
    return 0;
}

File: 1. Sum of cubes/code.java
public class code {
    public static int sumOfCubes(int n, int m) {
        int totalSum = 0;
        for (int i = n; i <= m; i++) {
            totalSum += i * i * i;
        }
        return totalSum;
    }

    public static void main(String[] args) {
        int n = 2, m = 4;
        System.out.println(sumOfCubes(n, m)); // Output should be 99
    }
}